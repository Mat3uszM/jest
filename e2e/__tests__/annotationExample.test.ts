/**
 * Copyright (c) Facebook, Inc. and its affiliates. All Rights Reserved.
 *
 * This source code is licensed under the MIT license found in the
 * LICENSE file in the root directory of this source tree.
 */

import {helper} from './annotationHelper';

test('failing snapshot example', () => {
  expect('two').toMatchSnapshot();
});

describe('nested', () => {
  test('failing example', () => {
    expect(10).toBe(1);
  });
});

test('passing helper', () => {
  expect(() => helper()).toThrow('Helper logged an error');
});

test('failing helper', () => {
  helper();
});
