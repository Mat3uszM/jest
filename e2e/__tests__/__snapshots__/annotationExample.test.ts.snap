// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`failing snapshot example 1`] = `"one"`;
